lib/cli.mjs

```diff
@@ -74,9 +74,14 @@ const helpText = `
         A pattern of files to ignore (in addition to those in "${DEFAULT_IGNORE_FILENAME}").
         Multiple can be set.
 
-      --fix
+      --fix [<mode>]
 
-        Automatically fix problems of certain rules.
+        Automatically fix problems of certain rules. The modes are as follows:
+
+          true         automatically fixes issues, same as "lax" (default)
+          false        does not fix any problems
+          lax          attempting to fix as much as possible even with syntax errors
+          strict       only fixing when there are no syntax errors
```

Comment:
@ybiquitous @Mouvedia 
[This commit](https://github.com/stylelint/stylelint/pull/8106/commits/f59eade747442f97d3257ad85954c20be8badbcb) modifies the behavior to accept only the following three patterns:

```shell
# If nothing is specified, meow treats it as true
--fix
--fix=lax
--fix=strict
```

If there are no issues with this behavior, I'll update the docs as well.

To support receiving values like the following:

```shell
--fix=true
--fix=false
```

It needs to be configured as follows:

```js
fix: {
  choices: [true, 'lax', 'strict', 'true', 'false'],
}
```

Reply:
Interesting; can you keep the `type` when using a `choices` that mixes booleans and strings?
If not, let's only use `choices` and drop the type.

Reply:
> If nothing is specified, `fix` will not be applied. In other words, the default is `undefined`.

I saw that you had to surround `fix` with `Boolean()`; would adding `default: false,` permit to use `fix` directly like before?

Reply:
I agree 4 cases should be provided as you commented 👍🏼 

- unspecified
- `--fix` (equals `--fix=lax`)
- `--fix=lax`
- `--fix=strict`

I don't think it's needed to support `--fix=true` and `--fix=false`.